FreeRTOS-Plus/WindowsSimulator/FreeRTOS-Kernel/FreeRTOSConfig.h

```diff
@@ -144,7 +144,7 @@
  * results in the wired network being used, while setting
  * configNETWORK_INTERFACE_TO_USE to 2 results in the wireless network being
  * used. */
-#define configNETWORK_INTERFACE_TO_USE      ( 1L )
```

Comment:
Perhaps this should go in the demo config?

Reply:
I'd agree with that, I'll remove it here